app/addons/documents/index-editor/components.react.jsx

```diff
@@ -24,19 +24,21 @@ function(app, FauxtonAPI, React, Stores, Actions, Components, beautifyHelper) {
   var indexEditorStore = Stores.indexEditorStore;
   var getDocUrl = app.helpers.getDocUrl;
 
-  var ToggleButton = React.createClass({
-
-    render: function() {
+  // global component
+  var StyledSelect = React.createClass({
```

Comment:
Yes, I want to tackle that immediatly after this one got merged, as I have at least one other component for that from older work: https://github.com/apache/couchdb-fauxton/blob/master/app/addons/documents/header/header.react.jsx#L27